float64 linear_vel_max      # Max linear velocity
float64 linear_acc_max      # Max linear acceleration
float64 angular_vel_max     # Max angular velocity
float64 angular_acc_max     # Max angular acceleration